x-pack/plugins/enterprise_search/server/lib/indices/fetch_connector_indices.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IScopedClusterClient } from '@kbn/core/server';
+import { fetchConnectors } from '@kbn/search-connectors';
+
+import { ElasticsearchIndexWithPrivileges } from '../../../common/types/indices';
+import { isNotNullish } from '../../../common/utils/is_not_nullish';
+import { fetchCrawlers } from '../crawler/fetch_crawlers';
+
+import { fetchIndexCounts } from './fetch_index_counts';
+import { fetchIndexPrivileges } from './fetch_index_privileges';
+import { TotalIndexData } from './fetch_indices';
+import { getIndexDataMapper, getUnattachedIndexData } from './utils/get_index_data';
+
+export const fetchConnectorIndices = async (
```

Comment:
the actual function is actually returning *non* connector indices, right?

Reply:
yes a rename is needed